pkg/docker/docker_test.go

```diff
@@ -629,3 +629,26 @@ func TestGetImageName(t *testing.T) {
 		}
 	}
 }
+
+func TestGetFullImageName(t *testing.T) {
+	type runtest struct {
+		imageName     string
+		serverAddress string
+		expected string
+	}
+	tests := []runtest{
+		{"test/image","test-harbor.io", "test-harbor.io/test/image:latest"},
```

Comment:
should add case like `{"test/image","test-harbor.io:3333", "test-harbor.io:3333/test/image:latest"}`

Reply:
ok